config/routes.rb

```diff
@@ -1,7 +1,10 @@
 Rails.application.routes.draw do
   devise_for :users
-  root to: 'pages#home'
-
+  # if current_user.renter == false 
```

Comment:
**suggestion:** Can we remove those commented lines please? Because we don't have access to `current_user` in the routes

Reply:
yup